docs/orchestration/concepts/artifacts.md

```diff
@@ -0,0 +1,88 @@
+
+Markdown artifacts render strings that can include [Github-flavored Markdown](https://github.github.com/gfm/) markup for formatting. 
+
+::: tip Artifacts render individually
+Note that each artifact you create in a task renders as an individual artifact in the Prefect UI. In other words, each call to `create_link()` or `create_markdown()` creates its own, separate artifact.
+
+Within a task, you may use these commands as many times as necessary, but they do not operate in the same manner as a `print()` command where you might string together multiple calls to add additional items to a report. 
+
+As a best practice, such as when using `create_markdown()` to create artifacts like reports or summaries, compile your message string separately, then pass to `create_markdown()` to create the full artifact.
+:::
+
+For more background on the design goals for the Artifacts API, see the [Introducing: The Artifacts API](https://medium.com/the-prefect-blog/introducing-the-artifacts-api-b9e5972db043) blog post and the [Great Expectations task](/api/latest/tasks/great_expectations.html).
+
+## UI
+
+In the Prefect UI, to view artifacts created by a flow run, navigate to a specific flow run, then click the **Artifacts** tab.
+
+![Screenshot showing the Artifacts tab for a flow run](/orchestration/concepts/artifacts_flowrun.png)
+
+You can navigate between the artifacts created for a flow run by clicking the dots or arrows above the artifacts.
+
+Each artifact is identified by task that created it.
+
+To view the artifacts created by a task run, navigate to the specific task run, then click the **Artifacts** tab.
+
+![Screenshot showing the Artifacts tab for a task run](/orchestration/concepts/artifacts_taskrun.png)
+
+You can navigate between the artifacts created for a task run by clicking the dots or arrows above the artifacts.
+
+## Creating Link Artifacts
+
+To create link artifacts, just import `create_link` from `prefect.artifacts`, then add a call to `create_link()` to any task orchestrated through a Prefect API. 
+
+Pass `create_link()` a string containing the absolute URL of the location to which you want to link. 
+
+```python
+from prefect import task, Flow
+from prefect.artifacts import create_link
+
+@task
+def send_artifact():
+    create_link("https://www.prefect.io/")
+```
+
+After the task runs, navigate to the UI’s Artifacts tab to see the output.
+
+## Creating Markdown Artifacts
+
+To create link artifacts, just import `create_markdown` from `prefect.artifacts`, then add a call to `create_markdown()` to any task orchestrated through a Prefect API. 
+
+Pass `create_markdown()` a string that will be rendered as an artifact. The string can contain any [Github-flavored Markdown](https://github.github.com/gfm/) markup including images, links, and tables. 
+
+Note that any images referenced in your markdown must be linked by absolute URL and served by an external web server. Linking to local files or by relative URL is not supported.
```

Comment:
instead of: "by absolute URL and served by an external web server."
I would phrase it something like: "by absolute URL of a publicly available image"
The reason: when I use a URL of an S3 object which is a publicly available image, then it's technically served from object storage rather than a web server, and this would also provide the info that it must be a publicly available image rather than e.g. served from a webserver in a company's VPN.

This is just a minor comment - you can keep the current formulation

Reply:
Good call